source/operators/rekognition/generic_data_lookup.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+###############################################################################
+# PURPOSE:
+#   JSON datasets can be precomputed and uploaded along with their associated
+#   media files to MIE. This lambda function is an operator that puts that
+#   precomputed data into DynamoDB. Don't forget to extend the Elasticsearch
+#   consumer (source/consumers/elastic/lambda_handler.py) if you want said data
+#   to be added to Elasticsearch so it can be searched and rendered on the MIE
+#   front-end.
+#
+# USAGE:
+#   JSON datasets must have the same filename as the associated video file. For
+#   example, "Demo Video 01.mp4" must have data file "Demo Video 01.json".
+#   Upload both files to the MIE dataplane bucket then add
+#   '"GenericDataLookup":{"Enabled":true}' to workflow configuration, like
+#   this:
+#
+#   curl -k -X POST -H "Content-Type: application/json" --data '{"Name":"MieCompleteWorkflow","Configuration":{"defaultVideoStage":{"GenericDataLookup":{"Enabled":true}}},"Input":{"Media":{"Video":{"S3Bucket":"'$DATAPLANE_BUCKET'","S3Key":"My Video.mp4"}}}}'  $WORKFLOW_API_ENDPOINT/workflow/execution
+#
+###############################################################################
+
+import os
+import json
+import urllib
+import boto3
+from MediaInsightsEngineLambdaHelper import OutputHelper
+from MediaInsightsEngineLambdaHelper import MasExecutionError
+from MediaInsightsEngineLambdaHelper import DataPlane
+
+operator_name = os.environ['OPERATOR_NAME']
+output_object = OutputHelper(operator_name)
```

Comment:
I think we're supposed to be using the Operation Helper now instead of the Output Helper class

Reply:
Done.